llm/mixtral/serve.yaml

```diff
@@ -34,6 +34,9 @@ setup: |
   pip list | grep megablocks || pip install megablocks
 
 run: |
+  # Remove the default nccl.conf which causes failure on Azure
+  sudo mv /etc/nccl.conf /etc/nccl.conf.bak || true
```

Comment:
What is this file for? Will this have any side-effects?

Reply:
Also is it a common issue for other workloads? Should we add it in `azure-ray.yml.j2` instead?

Reply:
This is a good point! This file is wrongly configured on Azure with multiple GPUs, causing the issue with the distributed workload using nccl. We should include this in the `azure-ray.yml.j2`.

Reply:
Fixed. PTAL.